src/components/TableOfContents/index.tsx

```diff
@@ -84,7 +84,7 @@ const TableOfContents: React.FC<IProps> = ({
       >
         <List {...outerListProps}>
           {!hideEditButton && editPath && (
-            <ListItem mb={2}>
+            <ListItem mb={4}>
```

Comment:
Lets try to minimize these style changes as much as we can in this migration

Reply:
😢 lol

Reply:
```suggestion
            <ListItem mb={2}>
```